README-DEV.md

```diff
@@ -137,7 +137,7 @@ dep-image-1-start.yml [compilers c1 c2]
 
 ##### Creation of `base-spack` and `dependency` images (dep-image-2-build.yml)
 
-In this workflow, given the specs for a given compiler, a `spack_packages` version, and a list of models for a future `model` matrix strategy, we seek to:
+In this workflow, given the specs for a given compiler, a `spack-packages` version, and a list of models for a future `model` matrix strategy, we seek to:
 
 * Check that a suitable `base-spack` image doesn't already exists. This would be one that has the same compiler and same version of `spack_packages`.
```

Comment:
```suggestion
* Check that a suitable `base-spack` image doesn't already exists. This would be one that has the same compiler and same version of `spack-packages`.
```